tests/test_5_mut.py

```diff
@@ -462,3 +463,26 @@ def test_sanitize():
             m.sanitize()
             assert_equal(err.getvalue().strip(),
                          'Warning: while appending section: 2 to parent: 0\nThe section first point should be parent section last point: \n        : X Y Z Diameter\nparent last point :[2.000000, 0.000000, 0.000000, 2.000000]\nchild first point :[2.000000, 1.000000, 0.000000, 2.000000]')
+
+
+def test_glia():
+    g = GlialCell()
+    assert_equal(g.cell_family, CellFamily.GLIA)
+
+    g = GlialCell(os.path.join(_path, 'astrocyte.h5'))
+    assert_equal(g.cell_family, CellFamily.GLIA)
+
+    g = GlialCell(Path(_path, 'astrocyte.h5'))
```

Comment:
What does this test purpose compared to test above (line 470) ? They seem to be identical.